lib/core/src/examples/easing_animator.rs

```diff
@@ -125,45 +170,49 @@ struct SubExample {
 
 impl SubExample {
     fn new<F>
-    ( mut event_loop   : &mut EventLoop
-    , graph_canvas     : Canvas
-    , animation_canvas : Canvas
-    , f                : &'static F
-    , origin_position  : Vector3<f32>
-    , target_position  : Vector3<f32>) -> Self
```

Comment:
As I know, this is properly formatted signature, not like in the new version. Or maybe the guidelines changed?

Reply:
@farmaazon I am not sure I got it. Isn't it complying to this rule?
```rust
pub fn new<Dom:Str>
( dom        : Dom
, logger     : Logger
, on_dirty   : OnDirty
, on_remove  : OnRemove
, on_replace : OnReplace
) -> Result<Self,Error> {
    ...
}
```

Reply:
Yes - the old version. I saw change adding the space before each comma.